Initialization/linux-initialization-2.md

```diff
@@ -454,9 +453,9 @@ ffff880000000000 - ffffc7ffffffffff (=64 TB) direct mapping of all phys. memory
 ...
 ```
 
-As you may see,  the virtual address space `ffff880000000000-ffffc7ffffffffff` is direct mapping of all physical memory. When the kernel wants to access all physical memory, it uses direct mapping.
+As explained above, the virtual address space `ffff880000000000-ffffc7ffffffffff` is direct mapping of all physical memory. When the kernel wants to access all physical memory, it uses direct mapping.
```

Comment:
As mentioned in another comment, those addresses are not up-to-date.

Reply:
> Thanks for the additions, I added a few nitpicks and requests.

Thank you for your review and feedback.